projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/input-docs/input-docs.component.ts

```diff
@@ -79,6 +79,16 @@ export class InputDocsComponent implements OnInit {
     // Use this.lastName.enable(); to re-enable the input.
   }
   `;
+
+  basicLengthExample: string = `
+  <go-input
+    [control]="name"
+    label="Your Name"
+    maxlength="10"
+    minlength="5">
```

Comment:
Not sure how much this matters, but since the type of these Inputs is `number` rather than `string`, should these bindings technically have brackets around them (i.e. `[maxlength]` and `[minlength]`)?